extensions/sql-migration/src/wizard/assessmentDetailsPage/assessmentDetailsPage.ts

```diff
@@ -30,7 +31,8 @@ export class AssessmentDetailsPage extends MigrationWizardPage {
 
 	constructor(
 		wizard: azdata.window.Wizard,
-		migrationStateModel: MigrationStateModel) {
+		migrationStateModel: MigrationStateModel,
+		private wizardController: WizardController) {
```

Comment:
Is there any specific reason we have kept this property private? just for understanding purposes.

Reply:
I generally follow the approach where if variable needs to be called only by internal functions, Make them restricted accessible.